iep-module-aws2/src/main/java/com/netflix/iep/aws2/AwsClientFactory.java

```diff
@@ -88,6 +95,18 @@ private void setIfPresent(
     }
   }
 
+  private void setRetriesIfPresent(Config cfg, ClientOverrideConfiguration.Builder builder) {
+    if (cfg.hasPath("retries")) {
```

Comment:
How likely do you think it is that we might want to have a way to more generically specify other aspects of the retry policy in the future? Perhaps a config structure like:

```
retry-policy {
  num-retries = 5
}
```

That would make it easier to extend later if we wanted to have some of the other parts configurable.

Reply:
> How likely do you think it is that we might want to have a way to more generically specify other aspects of the retry policy in the future?
I can see cases that might be needed (though I expect it to be rare). I like the idea of a config structure so we can easily extend it, if needed. Thanks! Will push the change.